packages/app/src/utils/API/nominationsAPI.js

```diff
@@ -29,6 +29,9 @@ const nominationsAPI = {
   syncNominations: function () {
     return axios.get(`${cleanURL(API_URL)}/api/syncnominations`);
   },
+  validateEmail: function (token) {
+    return axios.get(`${cleanURL(API_URL)}/api/confirmation/${token}`);
```

Comment:
I know this route already existed before this PR, but this should probably be a POST instead of a GET since it isn't returning any data.

Reply:
makes sense, will update the route too